salt/__init__.py

```diff
@@ -119,3 +119,6 @@ def __define_global_system_encoding_variable__():
 
 # This is now garbage collectable
 del __define_global_system_encoding_variable__
+
+# Import Salt's logging machinery
+import salt._logging.impl  # isort:skip  pylint: disable=unused-import
```

Comment:
What happens for older versions of pip? Not all OSes will have the newest version, right?

Reply:
Nothing, we hack around if we need to. On older versions we don't do anything.

Reply:
https://github.com/saltstack/salt/pull/60625/files#diff-721c42a3a37808f0a965af6afc0d1f17dc854895084f1071bbd00fe9dcfceed0R400
That module only exists on versions that need the work around.